cinema/models.py

```diff
@@ -76,17 +76,22 @@ class Meta:
 
 class Ticket(models.Model):
     movie_session = models.ForeignKey(
-        MovieSession, on_delete=models.CASCADE, related_name="tickets"
-    )
-    order = models.ForeignKey(
-        Order, on_delete=models.CASCADE, related_name="tickets"
+        MovieSession, on_delete=models.CASCADE,
+        related_name="tickets"
     )
+    order = models.ForeignKey(Order, on_delete=models.CASCADE,
+                              related_name="tickets")
     row = models.IntegerField()
     seat = models.IntegerField()
 
     @staticmethod
-    def validate_ticket(row, seat, cinema_hall, error_to_raise):
-        for ticket_attr_value, ticket_attr_name, cinema_hall_attr_name in [
+    def validate_ticket(row,
+                        seat,
+                        cinema_hall,
+                        error_to_raise):
```

Comment:
Ensure that the `error_to_raise` parameter in the `validate_ticket` method is always a subclass of `ValidationError`. This will prevent runtime errors if an incorrect error type is passed.